database/src/main/java/info/metadude/android/eventfahrplan/database/repositories/LecturesDatabaseRepository.kt

```diff
@@ -84,46 +84,48 @@ class LecturesDatabaseRepository(
-            lecture = lecture.copy(changedRecordingOptOut = cursor.getInt(CHANGED_RECORDING_OPTOUT).isChanged)
-            lecture = lecture.copy(changedRoom = cursor.getInt(CHANGED_ROOM).isChanged)
-            lecture = lecture.copy(changedSpeakers = cursor.getInt(CHANGED_SPEAKERS).isChanged)
-            lecture = lecture.copy(changedSubtitle = cursor.getInt(CHANGED_SUBTITLE).isChanged)
-            lecture = lecture.copy(changedTime = cursor.getInt(CHANGED_TIME).isChanged)
-            lecture = lecture.copy(changedTitle = cursor.getInt(CHANGED_TITLE).isChanged)
-            lecture = lecture.copy(changedTrack = cursor.getInt(CHANGED_TRACK).isChanged)
+
+            val lecture = Lecture(
+                    eventId = cursor.getString(EVENT_ID),
+                    abstractt = cursor.getString(ABSTRACT),
+                    date = cursor.getString(DATE),
+                    dateUTC = cursor.getLong(DATE_UTC),
+                    dayIndex = cursor.getInt(DAY),
+                    description = cursor.getString(DESCR),
+                    duration = cursor.getInt(DURATION),
+                    language = cursor.getString(LANG),
+                    links = cursor.getString(LINKS),
+                    recordingLicense = cursor.getString(REC_LICENSE),
+                    relativeStartTime = cursor.getInt(REL_START),
+                    room = cursor.getString(ROOM),
+                    roomIndex = cursor.getInt(ROOM_IDX),
+                    slug = cursor.getString(SLUG),
+                    speakers = cursor.getString(SPEAKERS),
+                    subtitle = cursor.getString(SUBTITLE),
+                    startTime = cursor.getInt(START),
+                    title = cursor.getString(TITLE),
+                    track = cursor.getString(TRACK),
+                    type = cursor.getString(TYPE),
+                    url = cursor.getString(URL),
+                    recordingOptOut = recordingOptOut,
+                    changedDay = cursor.getInt(CHANGED_DAY).isChanged,
+                    changedDuration = cursor.getInt(CHANGED_DURATION).isChanged,
+                    changedIsCanceled = cursor.getInt(CHANGED_IS_CANCELED).isChanged,
+                    changedIsNew = cursor.getInt(CHANGED_IS_NEW).isChanged,
+                    changedLanguage = cursor.getInt(CHANGED_LANGUAGE).isChanged,
+                    changedRecordingOptOut = cursor.getInt(CHANGED_RECORDING_OPTOUT).isChanged,
+                    changedRoom = cursor.getInt(CHANGED_ROOM).isChanged,
+                    changedSpeakers = cursor.getInt(CHANGED_SPEAKERS).isChanged,
+                    changedSubtitle = cursor.getInt(CHANGED_SUBTITLE).isChanged,
+                    changedTime = cursor.getInt(CHANGED_TIME).isChanged,
+                    changedTitle = cursor.getInt(CHANGED_TITLE).isChanged,
+                    changedTrack = cursor.getInt(CHANGED_TRACK).isChanged
+            )
```

Comment:
Please also apply this pattern to the repositories (`AlarmsDatabaseRepository#query`, `MetaDatabaseRepository#query`) which also use the `copy` method to have a consistent code base.